src/main/java/core/basesyntax/impl/WritingToNewFileImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.WritingToNewFile;
```

Comment:
It seems like the `WritingToNewFile` interface is not in the correct package. According to the checklist, it should be under the `service` package.